src/main/java/com/redhat/exhort/integration/providers/ProviderResponseHandler.java

```diff
@@ -227,7 +230,7 @@ public ProviderReport buildReport(
     return new ProviderReport().status(defaultOkStatus(getProviderName())).sources(reports);
   }
 
-  public Source buildReportForSource(
+  private Source buildReportForSource(
```

Comment:
@ruromero maybe consider in the future to change the access modifier to protected to let subclasses call it, if it will make sense or will be needed.

Reply:
Thanks for the suggestion, it can be changed as soon as it is necessary.